examples/showcase-curved-mockups/index.html

```diff
@@ -8,18 +8,19 @@
   </head>
   <body>
     <a-scene>
```

Comment:
[http://localhost:9000/examples/showcase-curved-mockups/](http://localhost:9000/examples/showcase-curved-mockups/)

camera regressed and cursor missing:

<img width="1363" alt="screenshot 2016-02-17 03 18 01" src="https://cloud.githubusercontent.com/assets/203725/13107970/313af2b0-d525-11e5-9fa8-c4a1d880e053.png">

<img width="1363" alt="screenshot 2016-02-17 03 08 28" src="https://cloud.githubusercontent.com/assets/203725/13107969/313aa990-d525-11e5-8782-43fc459cc24e.png">